README.md

```diff
@@ -30,20 +30,28 @@ The optional parameters are:
 11. ***METADATA***. This defines optional metadata to be set for the VM. Ex: { key1 = "val", key2 = "val2"}
 12. ***LABELS***. This defines key value pairs to set as labels when the VMs are created. Ex: { key1 = "val", key2 = "val2"} 
 13. ***STARTUP_COMMAND***. This defines the startup command to run when the VM starts up. Ex: python /usr/cp/train.py
-14. ***ORCHESTRATOR_TYPE***. This defines the Orchestrator type to be set up on the VMs. The current supported orchestrator types are 
+14. ***ORCHESTRATOR_TYPE***. This defines the Orchestrator type to be set up on the VMs. The currently supported orchestrator types are 
     -  __ray__: A Ray cluster is created using the MIG instances.
     -  __gke__: A private GKE cluster is created with private nodepool following the recommendations from the GKE team.
-15. ***GCS_MOUNT_LIST***. This defines the list of GCS buckets to mount. The format is `<bucket1>:</mount/path1>,<bucket2>:</mount/path2>`. For example: GCS_MOUNT_LIST=test-gcs-bucket-name:/usr/trainfiles
-16. ***NFS_FILESHARE_LIST***. This defines the list of NFS file shares to mount. The format is `</mount/path1>:<NFS fileshare type>,</mount/path2>:<NFS fileshare type>:<NFS fileshare size in GB>`. For example: NFS_FILESHARE_LIST=/usr/nfsshare1:BASIC_SSD
+15. ***NODE_POOL_0, NODE_POOL_1...***: When the Orchestrator type is set to GKE, this paramter allows to specify node pools. The format of each node pool is `<node_pool_name>:<min_node_count>:<max_node_count>`. Currently we expect all the node pools to be homogeneous. The `VM_TYPE` parameter is used for node pool machine type. `ACCELERATOR_TYPE` and `GPU_COUNT` is used for GPU configuration for each node pool.
+
+         For Example:
+         NODE_POOL_1=test-pool-1:2:4
```

Comment:
If we plan to also specify per-NodePool PlacementPolicy, what will be the syntax?
In general I think we are at the limit of expressing desired config using flat key=value settings. Right here we are trying to represent a nested data structure and inventing our own syntax.
Is this perhaps a hint that we should transition to something like .yaml?

Reply:
Ya, node pool is a complex data structure that we are introducing for the first time.
I am thinking, should we just let the user choose number of node pools ? Since we are saying the node pools are homogeneous, we can use the `INSTANCE_COUNT` as desired number of nodes in the node pool.
If we do that we can avoid complications here.